libsolidity/ast/Types.cpp

```diff
@@ -1808,6 +1808,23 @@ string ArrayType::toString(bool _short) const
 	return ret;
 }
 
+string ArrayType::humanReadableName() const {
```

Comment:
Style nitpick. We put the `{` on its own line

Reply:
fixed!